sci-misc/boinc/files/boinc.init

```diff
@@ -23,30 +23,45 @@ create_work_directory() {
 		chown "${USER}:${GROUP}" "${RUNTIMEDIR}"
```

Comment:
What error can happen when root tries to chown a directory that exists?
Oh! You thought of users having changed the config possibly setting a user or group that does not exist, right?
Good find! I never thought about that. Thanks!